src/partials/projects/projects.html

```diff
@@ -1 +1,166 @@
+                            <img
+                              class="about-me-img"
+                              src="/assets/img/projects/2_image_mobile_1x.webp"
+                              alt="Project 2"
+                              width="311"
+                              height="195"
+                              loading="lazy"
+                              />
+                        </picture>
+                    </div>
+                  </div>
+                </div>
+              </li>
+
+              <li class="project-list-item">
+                <div class="project-card">
+                  <div class="left-part">
+                    <div class="technologies">
+                      <div class="technology">#react</div>
+                      <div class="technology">#js</div>
+                      <div class="technology">#node js</div>
+                      <div class="technology">#git</div>
+                    </div>
+                    <h3>Programming Across Borders: Ideas, Technologies, Innovations</h3>
+                    <button class="project-btn" onclick="openInNewTab('https://https://github.com/project1')">See project</button>
+                  </div>
+                  <div class="right-part">
+                    <div class="project-img-wrapper">
+                        <picture>
+                            <source
+                               media="(max-width: 767.98px)"
+                               srcset="
+                                  /assets/img/projects/3_image_mobile_1x.webp 1x,
+                                  /assets/img/projects/3_image_mobile_2x.webp 2x
+                                "
+                            />
+                            <source
+                               media="(min-width: 768px)"
+                               srcset="
+                               /assets/img/projects/3_image_tab_desk_1x.webp 1x,
+                               /assets/img/projects/3_image_tab_desk_2x.webp 2x
+                                "
+                            />
+                            <img
+                              class="about-me-img"
+                              src="/assets/img/projects/3_image_mobile_1x.webp"
+                              alt="Project 1"
+                              width="311"
+                              height="195"
+                              loading="lazy"
+                              />
+                        </picture>
+                    </div>
+                  </div>
+                </div>
+              </li>
+          </ul>
+        </div>
+      </div>
+
+<div class="projects-swiper-btns">
+        <button
+          class="project-swiper-btn prev"
+          id="prevBtn"
+          aria-label="button to view the previous review"
+        >
+          <svg class="project-btn-icon" width="24" height="24">
+            <use href="../../assets/svg/icons.svg#icon-arrow-narrow-left-dark"></use>
```

Comment:
Please changre  href from
`<use href="../../assets/svg/icons.svg#icon-arrow-narrow-left-dark"></use>`
to 
`<use href="/assets/svg/icons.svg#icon-arrow-narrow-left-dark"></use>`